README.md

```diff
@@ -161,7 +161,7 @@ namespace App\Service;
 
 
 use GuzzleHttp\Client;
```

Comment:
@HypeMC We should also update the example so it uses Symfony's HTTP client instead. 😄

Reply:
Same here, out of scope.